lua/telescope/previewers/term_previewer.lua

```diff
@@ -225,7 +225,8 @@ previewers.new_termopen_previewer = function(opts)
           return
         end
 
-        vim.fn.chansend(term_id, termcode)
+        vim.fn.chansend(term_id, termcode .. (utils.is_windows() and "\r\n" or ""))
+        vim.api.nvim_buf_call(self.state.termopen_bufnr, function() vim.cmd("normal! G") end)
```

Comment:
you found that this was necessary on windows? just confirming.

Reply:
yes, working like a charm in Windows 11